### ISSUE TITLE
There is a bug in the DOSBox detection routine.

### ISSUE THREAD

**Issue by myrkraverk** [fone32/extenders#4]
In [dosbox.inc](https://github.com/fone32/extenders/blob/master/fone32/fapi/dosbox.inchttps://github.com/fone32/extenders/blob/master/fone32/fapi/dosbox.inc) the second signature string is incorrect.  It should be `DOSBox FakeBIOS v` with a space.  At least my copy of DOSBox has the full string `DOSBox FakeBIOS v1.0`.

**Comment by dcoshea** [comment]
Yes, the source for DOSBox 0.73 and 0.74 contains `"DOSBox FakeBIOS v1.0"`.
